pg-ragtime/project.clj

```diff
@@ -0,0 +1,27 @@
+(defproject com.github.igrishaev/pg2-ragtime "0.1.3-SNAPSHOT"
+
+  :description
+  "Ragtime migrations for PG2"
+
+  :plugins
+  [[lein-parent "0.3.8"]]
+
+  :dependencies
+  [[com.github.igrishaev/pg2-core]
+   [ragtime/ragtime]]
```

Comment:
You also need to add the `[dev.weavejester/ragtime "0.9.3"]` vector to the top-level `project.clj`, into the `:managed-dependencies` vector.